apps/condo/domains/news/components/TemplatesTabs.tsx

```diff
@@ -63,6 +63,9 @@ function getTabsWrap (): HTMLElement {
     return document.querySelector('.condo-tabs-nav-wrap')
 }
 
+/**
+ * @deprecated Use TemplatesSelect component instead. Same API, but renders to different control
```

Comment:
Maybe also add a todo mark of code removal here? Is that related to task DOMA-9331?

Reply:
Yep, 9331 is about getting rid of news-sharing feature flags